public/stylesheet.css

```diff
@@ -264,16 +276,24 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 40px;
+  width: 40px;
+  margin: 10px;
+  padding: 5px;
+  fill: var(--color-menu-icon);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-item:focus {
+  cursor: pointer;
+  height: 48px;
+  width: 48px;
+  margin: 6px;
```

Comment:
Interesting choice for a hover/focus effect. I would personally prefer something more similar to existing hover effects (none of which scale the hover target). E.g. change the color of the icon or invert it and add a background. What do you think? If you'd like to stick with what you have now, I think we should ask for input from other maintainers.

Reply:
I started with the inverse color effect. the problem is that it looks good in the buttons because they have borders. Without borders in the menu logos (which looks ugly) the other choice is to have a round "halo" to serve as background when inverting the color. It's never exactly the same as the buttons hover effect, so this one was faster.
I can go back to inverting the colors, if we are ok living with the circle color to serve as background.
Or any other idea to provide a background for the inverted icon.